clickhouse_search/models.py

```diff
@@ -600,6 +626,51 @@ class Meta:
         db_table = 'GRCh38/MITO/annotations_disk'
         engine = EmbeddedRocksDB(0, f'{CLICKHOUSE_DATA_DIR}/GRCh38/MITO/annotations', primary_key='key', flatten_nested=0)
 
+
+class BaseAnnotationsSv(BaseAnnotationsSvGcnv):
+    POPULATION_FIELDS = [
+        ('gnomad_svs', NamedTupleField([
+            ('af', models.DecimalField(max_digits=9, decimal_places=5)),
+            ('het', models.UInt32Field()),
+            ('hom', models.UInt32Field()),
+            ('id', models.StringField()),
+        ])),
+    ]
+    SV_TYPE_DETAILS = [(1, 'INS_iDEL'),(2, 'INVdel'),(3, 'INVdup'),(4, 'ME'),(5, 'ME:ALU'),(6, 'ME:LINE1'),(7, 'ME:SVA'),(8, 'dDUP'),(9, 'dDUP_iDEL'),(10, 'delINV'),(11, 'delINVdel'),(12, 'delINVdup'),(13, 'dupINV'),(14, 'dupINVdel'),(15, 'dupINVdup')]
+
+    algorithms = models.StringField(low_cardinality=True)
+    bothsides_support = models.BoolField(db_column='bothsidesSupport')
+    cpx_intervals = NestedField([
+        ('chrom', models.Enum8Field(return_int=False, choices=[(i+1, chrom) for i, chrom in enumerate(CHROMOSOMES)])),
+        ('start', models.UInt32Field()),
+        ('end', models.UInt32Field()),
+        ('type', models.Enum8Field(return_int=False, choices=BaseAnnotationsSvGcnv.SV_TYPES)),
+    ], db_column='cpxIntervals', null_when_empty=True)
+    end_chrom = models.Enum8Field(db_column='endChrom', return_int=False, choices=[(i+1, chrom) for i, chrom in enumerate(CHROMOSOMES)])
+    sv_source_detail = NestedField(
+        [('chrom', models.Enum8Field(return_int=False, choices=[(i+1, chrom) for i, chrom in enumerate(CHROMOSOMES)]))],
```

Comment:
it might be worth defining a `CHROMOSOME_CHOICES` constant since this definition seems to be copied like 5 times in this file now